### PR TITLE
Add support for Willis

### PR DESCRIPTION
**Pull request by krikun98**
Add support for the Willis (wireless Bruce).

PCB: https://github.com/krikun98/Willis
ZMK config: https://github.com/krikun98/willis-zmk-config
ZMK module: https://github.com/krikun98/willis-zmk-module

### CODE REVIEW

File: readme.org [hunk 1]
```diff
@@ -98,7 +98,7 @@ See [[#supported-keyboards][Supported Keyboards]] for details of supported keybo
 
 Specifies the ZMK board.
 
-For onboard controller keyboards (keyboards with an integrated controller), enter the keyboard name, e.g. ~ahokore~, ~ble_chiffre~, ~technikable~, ~zaphod~.
+For onboard controller keyboards (keyboards with an integrated controller), enter the keyboard name, e.g. ~ahokore~, ~ble_chiffre~, ~technikable~, ~zaphod~, ~willis~.
```
Comment (manna-harbour) [inline readme.org:101]: This is just a small number of examples, not a complete list, so it doesn't need to be added here. Also, they are in alphabetical order.

**Comment by manna-harbour** [comment]
Nice board, great name!

I've added one small file comment. Apart from that it's ready to merge, thanks!

File: config/willis.keymap [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Copyright 2021 Manna Harbour
```
Comment (manna-harbour) [inline config/willis.keymap:1]: Please update the dates in any files that are new or modified.

Comment (krikun98) [reply config/willis.keymap]: Updated, thank you.

File: readme.org (hunk 1 above)
Comment (krikun98) [reply readme.org]: Removed the example, thank you. I was just following the changes from the `technikable` commit.

Comment (manna-harbour) [reply readme.org]: Thanks! I thought I was keeping examples to 3, so I don't know why there was 4 there already!

**Comment by manna-harbour** [comment]
Thanks!

**Comment by manna-harbour** [comment]
Sample build run: https://github.com/manna-harbour/miryoku_zmk/actions/runs/11584527204

**Pull request merged**